api/wd/index.md

```diff
@@ -525,6 +525,38 @@ browser.evaluate(function fakeConfigData() {
 browser.evaluate('hash', function getLocationProp(prop) {
   return window.location[prop];
 });
+
+#### `browser.evaluateAsync(args..., function(args...))`
+
+The executed script is assumed to be asynchronous.
+
+Returns a Promise of the given function, invoked on the webdriver side (so you can not bind its `this` object or access context variables via lexical closure).
+If you provided `args`, testium will marshals them as JSON and passes them to the function in the given order.
+
+Note: NOT support Phantomjs
+
+##### Example: Fetch the data
+
+```js
+// Fetching data from the client
+browser.evaluate(() => {
+  return fetch('http://example.com/movies.json')
+    .then(response => response.json());
+});
```

Comment:
nice example - maybe skip the `{ return }` and just use implict return to make it super-pretty?

Reply:
@dbushong 
I keep the `return` just want to remind they don't forget to return a `Promise` :)

Reply:
Add the `return` in the async example